py/kubeflow/tf_operator/release.py

```diff
@@ -145,11 +145,11 @@ def build_operator_image(root_dir,
   commit = build_and_push_image.GetGitHash(root_dir)
 
   targets = [
-    "github.com/kubeflow/tf-operator/cmd/tf-operator.v1",
```

Comment:
if we want to continue using it then we should rename the folder to training_operator as well